src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -461,16 +480,42 @@
     }
   }
 
+  .topRowWrapper {
+    display: flex;
+    flex-direction: row;
+    gap: min(var(--space-lg), 50px);
+    position: sticky;
+    top: var(--space-lg);
```

Comment:
this is not doing anything, the element is `position: static` so top has no effect

Reply:
It has the position `sticky`, not `static`, but since we don't want the whole screen to scroll (like you mentioned earlier), I will remove `sticky` and `top` entirely

Reply:
yeah, sorry, I was playing around and I remove the sticky incorrectly so it was just static for me when I checked this :see_no_evil: